### DISCUSSION TITLE
Assessoria Contábil de Contabilidade para PJ e PF

### DISCUSSION THREAD

**Discussion by felquis** [DyegoCosta/trabalhando-remoto#157] [General]
Oi pessoal, como tem sido sua experiência com escritórios de contabilidade para administrar sua PJ?

Da uma olhada na minha experiência...

- Tiram todas as minhas dúvidas
- Me instruem como devo agir como PJ
- Aprendem junto comigo enquanto vou me aventurando em diferentes tipos de serviços digitais
- Me lembram de minhas obrigações
- Revisam os impostos que pago visando me enquadrar na tributação mais apropriada pagando menor imposto possível dentro da legalidade.
- No fim das contas acabo aprendendo um bocado pois também é necessário entender alguns fundamentos. 

Não conseguiria viver sem uma assessoria contábil por que é conveniência demais e não tenho o tempo para estudar todas as nuances contábeis.

E, estão com um programa de indicações, então indicando um de vocês, eles me darão um descontinho na mensalidade. Então interessados me mandem um oi bele? Abraço.

**Comment by jonaswebdev** [comment]
Oi [felquis](https://github.com/felquis), bem interessante. Qual escritório de Assessoria Contábil vc trabalha?

**Reply by felquis** [reply]
Há algumas informações de contato aqui https://www.atlacontabilidade.com.br/ espero que te ajude.

**Reply by jonaswebdev** [reply]
Vou checar, obrigado
